server_installation/topics/operator/external-database.adoc

```diff
@@ -0,0 +1,61 @@
+
+=== External database support
+
+Enabling external Postgresql database support requires creating `keycloak-db-secret` in the following form (note that values are Base64 encoded, see https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets[Kubernetes Secrets manual]):
+
+.`keycloak-db-secret` Secret
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+    name: keycloak-db-secret
+    namespace: keycloak
+stringData:
+    POSTGRES_DATABASE: <Database Name>
```

Comment:
Why not using the same DB_ prefixed variables as recommended in https://github.com/keycloak/keycloak-containers/tree/master/server ? Having to configure a different set of variables for the DB could be error prone.

Reply:
@fidgi The proposal to use [DB_ prefixed variables](https://github.com/keycloak/keycloak-containers/tree/master/server#environment-variables) LGTM. Could you file a new Keycloak Operator JIRA RFE to have this implemented in the future?

(the current documentation is describing the current variable names. From a brief look at the code, looks besides the variable names change in the documentation, also some variable names change would need to be done in the operator code itself, e.g. in [recognized constants](https://github.com/keycloak/keycloak-operator/blob/master/pkg/model/constants.go#L29). Since docs change itself wouldn't be sufficient, this might be a candidate for a future RFE / change)

@slaskawi WDYT?

Reply:
Thanks for the suggestion @fidgi @iankko !

We already have a change request around this in [KEYCLOAK-12040](https://issues.redhat.com/browse/KEYCLOAK-12040). Please follow up using that JIRA.

And of course, @iankko is correct here - the documentation describes the current approach. Supporting other databases is a feature request.